### ISSUE TITLE
Getting Fatal Exception: java.lang.NullPointerException

### ISSUE THREAD

**Issue by varun514** [ganfra/MaterialSpinner#153]
Fatal Exception: java.lang.NullPointerException: Attempt to invoke virtual method 'java.lang.String java.lang.Object.toString()' on a null object reference
       at android.widget.ArrayAdapter.createViewFromResource(ArrayAdapter.java:457)
       at android.widget.ArrayAdapter.getDropDownView(ArrayAdapter.java:505)
       at fr.ganfra.materialspinner.MaterialSpinner$HintAdapter.buildView(MaterialSpinner.java:786)
       at fr.ganfra.materialspinner.MaterialSpinner$HintAdapter.getDropDownView(MaterialSpinner.java:774)
       at android.widget.Spinner$DropDownAdapter.getDropDownView(Spinner.java:1007)
       at android.widget.Spinner$DropDownAdapter.getView(Spinner.java:1003)
       at android.widget.Spinner.measureContentWidth(Spinner.java:869)
       at android.widget.Spinner$DropdownPopup.computeContentWidth(Spinner.java:1240)
       at android.widget.Spinner$DropdownPopup.show(Spinner.java:1266)
       at android.widget.Spinner.performClick(Spinner.java:792)
       at android.view.View.performClickInternal(View.java:7218)
       at android.view.View.access$3800(View.java:824)
       at android.view.View$PerformClick.run(View.java:27719)
       at android.os.Handler.handleCallback(Handler.java:883)
       at android.os.Handler.dispatchMessage(Handler.java:100)
       at android.os.Looper.loop(Looper.java:228)
       at android.app.ActivityThread.main(ActivityThread.java:7782)
       at java.lang.reflect.Method.invoke(Method.java)
       at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:492)
       at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:981)
       
       
This issue is coming for me and it is not producible. 
Basically I am attaching the materialSpinner with a null Array<String> adapter and after data comes from api then adapter is updated.
